open-apparel-registry/open-apparel-registry
# Upgrade Django to 2.2 and upgrade Django-related libraries to latest versions

## Overview

- Upgrade Django to 2.2
- Upgrade some django-related libraries to their latest versions

I dd not upgrade libraries related to dedupe or batch processing, as I don't know how easy it is to test regressions there.

Connects #455 

## Demo

![Screen Shot 2019-07-16 at 8 15 23 PM](https://user-images.githubusercontent.com/4165523/61338222-75bb7380-a806-11e9-842a-3ab74f8bd33e.png)

## Testing Instructions

- `vagrant destroy` then `./scripts/setup` and verify that everything sets up as it should
- `./scripts/resetdb` and `./scripts/server`
- visit the app on 6543 and verify that the following stuff works:
    - registering (and getting registration email)
    - signing in
    - uploading a list
    - searching for facilities
    - saving profile updates & generating API tokens
- process the list you uploaded using `batch_process` and verify that that still works
- visit your list in the UI and confirm or reject some facility matches if any are available, verifying that that works and also that you can remove list items
- visit the API docs and verify that they load and that you can use them to test the various endpoints

## Checklist
